app/views/content_pages/_embedded_video.html.slim

```diff
@@ -0,0 +1,15 @@
+ 
+.govspeak-embed-container class="light-grey-background"
+  h2.govuk-heading-l class="govuk-!-margin-top-2" Video: #{@model.video_title}
+  div class='embed-container'
+    - if @model.video_provider == 'vimeo'
+      iframe.govspeak-embed-video allow=("autoplay; fullscreen; picture-in-picture; accelerometer; clipboard-write; encrypted-media; gyroscope;") allowfullscreen="" frameborder="0" src="https://player.vimeo.com/video/#{@model.video_id}?enablejsapi=1&amp;origin=#{ENV['DOMAIN']}" style="position:absolute;top:0;left:0;width:100%;height:100%;"  script src="https://player.vimeo.com/api/player.js"
+    - elsif @model.video_provider == 'youtube' 
+      iframe.govspeak-embed-video allow=("autoplay; fullscreen; picture-in-picture; accelerometer; clipboard-write; encrypted-media; gyroscope;") allowfullscreen="" frameborder="0" src="https://wwww.youtube.com/embed/#{@model.video_id}?enablejsapi=1&amp;origin=#{ENV['DOMAIN']}" style="position:absolute;top:0;left:0;width:100%;height:100%;"
```

Comment:
I think it would also be good to build the Vimeo and YouTube URLs in the model too, a method for each.